### PR TITLE
Updated README.md and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by patel-zeel**
Updated guidelines to follow once we move from scripts to notebooks workflow.

@murphyk Can you please review these guidelines?

### CODE REVIEW

**Comment by nalzok** [comment]
I think it's better to make the exception handler more specific, as suggested by [pylint](https://vald-phoenix.github.io/pylint-errors/plerr/errors/exceptions/W0703.html). A bare `except:` will catch exceptions raised during importing the packages, which might hide some errors.

```python
try:
    import tensorflow as tf
except ModuleNotFoundError:
    %pip install tensorflow
    import tensorflow as tf
```

**Comment by patel-zeel** [comment]
That's a good catch! We should systematically move to your suggested version for all the notebooks. I have opened an issue at #802. Would you like to take this up?

**Comment by nalzok** [comment]
Sure. I will take care of that. Should I just push to master, or open a PR first?

**Comment by patel-zeel** [comment]
@nalzok Since this change will modify all the notebooks, a PR will be best. We can associate it with the opened issue and upon resolution close the issue.

File: README_new.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+This is work in progress, so expect rough edges!
+(Some demos use code from our companion [JAX State Space Library](https://github.com/probml/JSL).)
+ 
+
+## Running the notebooks
+
+The notebooks needed to make all the figures are available at the following locations.
+
+* [Volume 1 figure notebooks](https://github.com/probml/pyprobml/tree/master/notebooks/book1)
+* [Volume 2 figure notebooks](https://github.com/probml/pyprobml/tree/master/notebooks/book1). (Note: volume 2 is not finished yet.)
```
Comment (murphyk) [inline README_new.md:16]: book1 -> book2

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # How to Contribute
 Kevin Murphy and Mahmoud Soliman.
```
Comment (murphyk) [inline CONTRIBUTING.md:2]: Feel free to add your names :)

File: README_new.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+```
+%run pyprobml/scripts/kf_tracking_demo.py # colab
+python3 pyprobml/scripts/kf_tracking_demo.py # locally
+```
+
+
+## GCP, TPUs, and all that
+
+When you want more power or control than colab gives you, you should get a Google Cloud Platform (GCP) account, and get access to a TPU VM. You can then use this as a virtual desktop which you can access via ssh from inside VScode. We have created [various tutorials on Colab, GCP and TPUs](https://github.com/probml/probml-notebooks/blob/main/markdown/colab_gcp_tpu_tutorial.md) with more information.
```
Comment (murphyk) [inline README_new.md:62]: stale URL

File: README_new.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+* Option 2
+
+Download [requirements.txt](https://github.com/probml/pyprobml/blob/master/requirements.txt) locally to your path and run
+
+```bash
+pip install -r requirements.txt
+```
+
+## Running scripts for vol 2
```
Comment (murphyk) [inline README_new.md:48]: Remove this section on 'scripts for vol 2'. (If there are any notebooks that need JSL, they can install it locally for that colab.)

File: README_new.md (hunk 1 above)
Comment (patel-zeel) [reply README_new.md:16]: Done

File: CONTRIBUTING.md (hunk 2 above)
Comment (patel-zeel) [reply CONTRIBUTING.md:2]: Done :)

File: README_new.md (hunk 3 above)
Comment (patel-zeel) [reply README_new.md:62]: Fixed.

**Comment by patel-zeel** [comment]
@murphyk I have resolved all the comments.

**Pull request merged**